### PR TITLE
feat: add btop spec

### PR DESCRIPTION
**Pull request by ben5vik**

### CODE REVIEW

**Comment by ben5vik** [comment]
i am confused that i get prettier errors

File: src/btop.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      name: ["--help", "-h"],
+      description: "Shows help for btop",
+    },
+    {
+      name: ["--low-color", "-lc"],
+      description: "Disables truecolor, converts 24-bit colors to 256-color",
+    },
+    {
+      name: ["--tty_on", "-t"],
+      description: "Forces ON tty mode, max 16 colors and tty friendly graph symbol",
```
Comment (fedeci) [inline src/btop.ts]:
```suggestion
      description:
        "Forces ON tty mode, max 16 colors and tty friendly graph symbol",
```

File: src/btop.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      name: ["--tty_off", "+t"],
+      description: "Forces OFF tty mode",
+      exclusiveOn: ["--tty_on"],
+    },
+    {
+      name: ["--preset", "-p"],
+      description: "Start with preset",
+      args: {
+        name: "id",
+        suggestions: ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9", ],
```
Comment (fedeci) [inline src/btop.ts]:
```suggestion
        suggestions: ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"],
```

File: src/btop.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      },
+    },
+    {
+      name: "--utf-force",
+      description: "Force start even if no UTF-8 locale was detected",
+    },
+    {
+      name: "--debug",
+      description: "Starts in DEBUG mode: shows microsecond timer for information collect and screen draw functions and sets loglevel to DEBUG",
```
Comment (fedeci) [inline src/btop.ts]:
```suggestion
      description:
        "Starts in DEBUG mode: shows microsecond timer for information collect and screen draw functions and sets loglevel to DEBUG",
```

File: src/btop.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    {
+      name: ["--version", "-v"],
+      description: "Shows the version of btop",
+    },
+  ],
+  };
```
Comment (fedeci) [inline src/btop.ts]:
```suggestion
};
```

File: src/btop.ts [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  export default completionSpec;
```
Comment (fedeci) [inline src/btop.ts]:
```suggestion
export default completionSpec;
```

**Review by fedeci — APPROVED** [review]
Thanks! I'm committing the required prettier changes, I am not sure why it isn't working correctly, have you installed npm deps?

**Comment by ben5vik** [comment]
> Thanks! I'm committing the required prettier changes, I am not sure why it isn't working correctly, have you installed npm deps?

I think I am pretty sure I did

**Comment by fedeci** [comment]
Yeah, it is now working in fact!

**Pull request merged**